WebContent/index.jsp

```diff
@@ -1,29 +1,34 @@
 <%@ page language="java" contentType="text/html; charset=ISO-8859-1"
-    pageEncoding="ISO-8859-1"%>
+	pageEncoding="ISO-8859-1"%>
 <%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
 <%@ page import="com.sundar.studentmanagement.student.vo.*"%>
-    
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+
+<!DOCTYPE html>
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
 <title>Student Management</title>
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<link href="./plugins/bootstrap.css" rel="stylesheet" />
-<script type="text/javascript" src="./plugins/bootstrap.min.js"></script>
-<script type="text/javascript" src="./plugins/jquery.2.2.3.min.js"></script>
-<script type="text/javascript" src="./plugins/ajax.js"></script>
+
+
+<!--  CSS and JS imports -->
+<jsp:include page="/jsp/common/allImports.jsp" />
+
+
 </head>
+
+
 <body>
-<nav class="navbar navbar-inverse">
-	<div class="container-fluid">
-		<div class="navbar-header">
-			<a class="navbar-brand" href="./././index.jsp">Student Management</a>

```

Comment:
what is ./././index.jsp ??  😡  dont import like this
Use 
< a class="navbar-brand" href='**/student_management-MVC-using-ajax/**' >Student Mgmnt < /a >

or --to get the project name (contextPath) dynamically use **<c:url value="/">**
< a class="navbar-brand" href='<c:url value="/" />' >Student Mgmnt < /a >